<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multigrid</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>620</width>
      <height>620</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_init">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_rand">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Randomize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_gol">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Game of Life - Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_path">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pathfinder - Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>